feat(kibbeh): updated hungarian translation

KristofKekesi opened the pull request.

amitojsingh366 commented:
please follow the proper pattern for commit messages *(💡TIP: you can use `yarn commit` or `yarn commit:signed` and answer the questions asked to generate a properly formatted one for you!)*

to fix your most recent commit you can run the command `git commit --amend -m "properly formatted commit message here"`

to fix your older/multiple commits please take a look at [this article](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages)

KristofKekesi commented:
@amitojsingh366 I'm using the online editor so I cannot run yarn commands (and I don't know what yarn is)

I've changed the PR's title to ´feat(kibbeh): updated hungarian translation´.

amitojsingh366 commented:
> @amitojsingh366 I'm using the online editor so I cannot run yarn commands (and I don't know what yarn is)
> 
> I've changed the PR's title to ´feat(kibbeh): updated hungarian translation´.

it's not the title of the pr that's causing the problem, it's the commit message
<img width="272" alt="Screen Shot 2021-05-06 at 2 22 06 AM" src="https://user-images.githubusercontent.com/35400192/117221411-dcfa4b80-ae11-11eb-9e21-2c7dd3859625.png">
you are not following the format *(`type(scope): message`)* and the only way i know to fix this is using the command line...
if you don't have access to one then ig you will have to make a new pr following the format. *(btw your pr title is exactly what your commit message is supposed to look like)*
